draft-demarco-oauth-status-attestations.md

```diff
@@ -395,6 +395,7 @@ the same position index to which it is related, eg: _[requestAboutA, requestAbou
 set to "status-assertion+{jwt,cwt}" or "status-assertion-error+{jwt,cwt}", depending by the object type.
 - The corresponding entry in the response MUST be of the same data format as requested. For example,
 if the entry in the request is "jwt", then the entry at the same position in the response MUST also be "jwt".
+- The corresponding entry in the response MUST NOT contain the intended audience or any information regarding the Verifier to whom the assertion may be presented.
```

Comment:
```suggestion
- The corresponding entry in the response MUST NOT contain any
information regarding the Verifier to whom it may be presented,
such as the Verifier identifier as the intended audience.
```